src/components/organisms/shared/OakPagination/OakPagination.tsx

```diff
@@ -1,5 +1,6 @@
-import React, { RefObject, useState } from "react";
+import React, { RefObject, useEffect, useState } from "react";
 import styled, { css } from "styled-components";
+import { NextRouter } from "next/router";
```

Comment:
Should router live in the component library? I'm not sure how intertwined it is but feels like it should be decoupled and use callbacks to handle the routing logic in OWA

Reply:
Decoupled the router from the component ✅